map/set-initial-map-location/README.metadata.json

```diff
@@ -1,25 +1,39 @@
 {
     "category": "Map",
-    "description": "Display a map centered at a latitude and longitude plus zoom level.",
+    "description": "Display a basemap centered at an initial location and scale.",
     "ignore": false,
     "images": [
         "SetInitialMapLocation.png"
     ],
     "keywords": [
-        "ArcGISMap",
+        "LOD",
+        "basemap",
+        "center",
+        "envelope",
+        "extent",
+        "initial",
+        "lat",
+        "latitude",
+        "level of detail",
+        "location",
+        "long",
+        "longitude",
+        "scale",
+        "zoom level",
         "Basemap",
-        "Basemap.Type",
+        "Map",
         "MapView"
     ],
-    "redirect_from": "/java/latest/sample-code/set-initial-map-location.htm",
+    "redirect_from": [
+        "/java/latest/sample-code/set-initial-map-location.htm"
+    ],
     "relevant_apis": [
-        "ArcGISMap",
         "Basemap",
-        "Basemap.Type",
+        "Map",
```

Comment:
this should be ArcGISMap for Java, same with then for the keywords

Reply:
In some instances it looks like the Readme still says `Map`, so I'll update the Readmes themselves and run the script again